// brasilapi-js/dist/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    var desc = Object.getOwnPropertyDescriptor(m, k);
    if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
      desc = { enumerable: true, get: function() { return m[k]; } };
    }
    Object.defineProperty(o, k2, desc);
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
    Object.defineProperty(o, "default", { enumerable: true, value: v });
}) : function(o, v) {
    o["default"] = v;
});
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (k !== "default" && Object.prototype.hasOwnProperty.call(mod, k)) __createBinding(result, mod, k);
    __setModuleDefault(result, mod);
    return result;
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.taxes = exports.registerBr = exports.pix = exports.ncm = exports.isbn = exports.ibge = exports.fipe = exports.holidays = exports.ddd = exports.cptec = exports.cvm = exports.cnpj = exports.cepV1 = exports.cep = exports.bank = exports.request = void 0;
const bank = __importStar(require("./routes/bank"));
exports.bank = bank;
const cep = __importStar(require("./routes/cep"));
exports.cep = cep;
const cepV1 = __importStar(require("./routes/cepV1"));
exports.cepV1 = cepV1;
const cnpj = __importStar(require("./routes/cnpj"));
exports.cnpj = cnpj;
const cvm = __importStar(require("./routes/cvm"));
exports.cvm = cvm;
const ddd = __importStar(require("./routes/ddd"));
exports.ddd = ddd;
const holidays = __importStar(require("./routes/holidays"));
exports.holidays = holidays;
const cptec = __importStar(require("./routes/cptec"));
exports.cptec = cptec;
const fipe = __importStar(require("./routes/fipe"));
exports.fipe = fipe;
const ibge = __importStar(require("./routes/ibge"));
exports.ibge = ibge;
const isbn = __importStar(require("./routes/isbn"));
exports.isbn = isbn;
const ncm = __importStar(require("./routes/ncm"));
exports.ncm = ncm;
const pix = __importStar(require("./routes/pix"));
exports.pix = pix;
const registerBr = __importStar(require("./routes/registerBr"));
exports.registerBr = registerBr;
const taxes = __importStar(require("./routes/taxes"));
exports.taxes = taxes;
const request = __importStar(require("./service/api.service"));
exports.request = request;
